scheduler/src/cook/mesos/util.clj

```diff
@@ -89,6 +89,16 @@
                 x))
             m))
 
+(defn entity->map
```

Comment:
Should we add unit tests with nested entities/components and verify that the returned map is what we expect?